BLSQ/iaso
# WC2-305: beneficiaries fixes

- location of a beneficiary is not present on the dashboard, we have the org unit but no geo location of the instance
- multi duplicates links in beneficiaries list is not working
- launch a search in beneficiaries list remove the selected beneficiary type
- beneficiaries default order should be recent first

Related JIRA tickets :WC2-306

## Self proofreading checklist


## Changes

- display submissions locations not only org units
- allow to switch between org unit locations or submissions locations
- fix routing issues
- fetch all the entities locations to display it on the map
- improve a bit entities api

## How to test

- Make sure you activate SHOW_BENEFICIARY_TYPES_IN_LIST_MENU account feature flag
- Make also sure you have entities with locations and or org unit with locations
- open entities for a specific beneficiary type
- try the map tab, you should have all the possible locations and switch between submissions or org unit locations
- switch from one type to another and make sure that search is relevant

## Print screen / video

https://github.com/BLSQ/iaso/assets/12494624/e0ac1acf-66e3-423c-8550-4344d074f172



## Notes

Call to entities API to fetch locations is very slow, i made a dedicated ticket for that [WC2-307](https://bluesquare.atlassian.net/browse/WC2-307)


[WC2-307]: https://bluesquare.atlassian.net/browse/WC2-307?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
